src/tests/mapUtils.test.ts

```
@@ -1,7 +1,13 @@
 import { LayerInfo } from '../data/mapData'
-import { mapRegionSelected, getActiveLayers } from '../utils/mapUtils'
-import { RegionOption } from '../types/RegionDataTypes'
-import { MapGeoJSONFeature, LngLat } from 'maplibre-gl'
+import {
+  createPolygonClickHandler,
+  createPolygonHoverHandler,
+  getActiveLayers,
+  mapRegionSelected,
+} from '../utils/mapUtils'
+import { Map, MapGeoJSONFeature, MapLayerMouseEvent } from 'maplibre-gl'
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)